src/config.ts

```diff
@@ -0,0 +1,93 @@
+import { ConfigurationScope, ConfigurationTarget, workspace } from 'vscode';
+import { VSCodeConfigurations } from './config-utils';
+
+export interface Style {
```

Comment:
This is actually [DecorationRenderOptions](https://code.visualstudio.com/api/references/vscode-api#DecorationRenderOptions) if I see it correctly. Currently not all are in the config for autocompletion in the settings, but if this class can be used directly, I would suggest that?

Reply:
In fact, I used a automatic generator from json-schema to TS 😅 . So yeah, I should've used `DecorationRenderOptions`.